static/src/javascripts/projects/commercial/modules/creatives/page-skin.js

```diff
@@ -110,16 +113,12 @@ const pageSkin = (): void => {
         }
     };
 
-    const repositionSkin = (): void => {
-        const header = document.querySelector('.new-header');
-        const footer = document.querySelector('.l-footer');
-        const topBannerAd = document.querySelector('.ad-slot--top-banner-ad');
-
-        if (truskinRendered && header && topBannerAd && footer) {
-            repositionTruskin(header, footer, topBannerAd);
+    const repositionSkins = (): void => {
+        if (truskinRendered && hasPageSkin) {
+            repositionTruskin();
         }
         // This is to reposition the Page Skin to start where the navigation header ends.
-        if (!truskinRendered && hasPageSkin && isInAUEdition) {
+        if (pageskinRendered && hasPageSkin && isInAUEdition) {
             repositionPageSkin();
         }
     };
```

Comment:
Is it worth adding any sort of logic here to handle if neither repositioning occurs? Maybe a `console.error` or `console.log` just to flag up that we tried to reposition but failed to call either event?

Reply:
Trying to think if it's an error if repositioning does not occur? If the conditions are not meant it could either mean
- The page doesn't have a `Truskin` irrespective of location or
- The page doesn't have a `Pageskin` in Australia
In both these cases it just means we don't want to perform any repositioning logic although we keep listening to `'window:throttledScroll'` event. Thinking about it, it would be better to attach the scroll event if either of truskin/pageskin exist instead of all times.

Reply:
Have now restricted the 'window:throttledScroll' to occur at least if there is a page skin on the page